src/components/Blogs/index.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { render, screen } from "@testing-library/react";
+import Blogs from "./index";
+import TestRenderer from "react-test-renderer";
+const { act } = TestRenderer;
+
+global.fetch = jest.fn(() =>
+  Promise.resolve({
+    json: () =>
+      Promise.resolve([
+        {
+          title: {
+            rendered: "You can find safe place for yourself here",
+          },
+          jetpack_featured_media_url:
+            "https://safeplace102505649.files.wordpress.com/2020/11/womentogether.jpg",
+          id: 78,
+        },
+      ]),
+  })
+);
+
+describe("Blogs", () => {
+  it("renders correctly and matches the snapshot", () => {
+    const tree = renderer.create(<Blogs />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("fetches and renders mock data correctly", async () => {
+    await act(async () => render(<Blogs />));
```

Comment:
it seems like it works but you only need one test, and that's the snapshot test, and so this rendering should be moved into the snapshot test

also see: 
![Uploading image.png…]()